src/views/Header.js

```diff
@@ -71,6 +76,24 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
     return this._getHeaderTitleString(lastScene);
   }
 
+  _getBackButtonTitleStyle(scene: NavigationScene): ?Style {
+    const lastScene = this._getLastScene(scene);
+    if (!lastScene) {
+      return null;
+    }
+    const {
+      headerBackTitleStyle,
```

Comment:
I've now added [a commit](https://github.com/react-community/react-navigation/pull/1083/commits/54f7eab34f7760d0349dbd0af67184125a0fbf6c) that applies both styles.